src/components/dashboard/datasets-accordion.js

```diff
@@ -1,22 +1,34 @@
 import { CodeBlock } from '@/components'
+import { useXarrayDatasetRepr } from '@/lib/endpoints'
 import { AddIcon, MinusIcon } from '@chakra-ui/icons'
 import {
   Accordion,
   AccordionButton,
   AccordionItem,
   AccordionPanel,
+  Alert,
+  AlertIcon,
   Box,
+  Skeleton,
 } from '@chakra-ui/react'
-import React from 'react'
 
 export const DatasetCard = ({ dataset }) => {
   const parts = dataset.split('/')
   const name = parts[parts.length - 1]
+  const { repr, reprError, isLoading } = useXarrayDatasetRepr(`${dataset}df`)
+
+  if (reprError)
+    return (
+      <Alert status='error' align='center' justifyContent='center'>
+        <AlertIcon />
+        {`Status Code: ${reprError.status}`} - {reprError.info?.detail}
+      </Alert>
```

Comment:
@katamartin, it turns out that our `useSWR`'s `JSONFetcher` returns an Error object with a bit of useful information

https://github.com/pangeo-forge/pangeo-forge.org/blob/8143745e7a8d6d32c6a03a8290e9465627f7b824/src/lib/fetchers.js#L8-L13

i ended up taking advantage of this and got rid of the `/api/repr/xarray` API route. So, this change ensures the errors are properly propagated to the client.

Reply:
<img width="1288" alt="Screen Shot 2022-09-10 at 10 14 27 PM" src="https://user-images.githubusercontent.com/13301940/189512117-c9b5fbe0-3b6d-48a2-84ca-f6e126ccab12.png">